react-components/toolbar-filter/step6.md

```diff
@@ -1,95 +1,13 @@
-        onSelect={this.onSelect}
-        type=&quot;Risk&quot;
-        selections={filters.risk}
-        options={riskOptions}
-      /&gt;
-    &lt;/DataToolbarItem&gt;
-  &lt;/DataToolbarGroup&gt;
-</pre>
-
-3) Replace each of the three `DataToolbarItems` inside the `DataToolbarToggleGroup` with a `DataToolbarFilter`. Pass the appropriate state managed filters array as the `chips` property, an appropriate label as the `categoryName` property, and the onDelete function as the `deleteChip` property. 
+1) Add the `DemoTable` component below the `DataToolbar` and pass it the status and risk filters as props.
 
 <pre class="file" data-target="clipboard">
-  &lt;DataToolbarFilter chips={filters.name} deleteChip={this.onDelete} categoryName=&quot;Name&quot;&gt;
-    &lt;Input id=&quot;searchInput&quot; search={this.onSearch} ariaLabel=&quot;Search input&quot; /&gt;
-  &lt;/DataToolbarFilter&gt;
-  &lt;DataToolbarGroup variant=&quot;filter-group&quot;&gt;
-    &lt;DataToolbarFilter chips={filters.status} deleteChip={this.onDelete} categoryName=&quot;Status&quot;&gt;
-      &lt;CheckboxSelect
-        onSelect={this.onSelect}
-        type=&quot;Status&quot;
-        selections={filters.status}
-        options={statusOptions}
-      /&gt;
-    &lt;/DataToolbarFilter&gt;
-    &lt;DataToolbarFilter chips={filters.risk} deleteChip={this.onDelete} categoryName=&quot;Risk&quot;&gt;
-      &lt;CheckboxSelect
-        onSelect={this.onSelect}
-        type=&quot;Risk&quot;
-        selections={filters.risk}
-        options={riskOptions}
-      /&gt;
-    &lt;/DataToolbarFilter&gt;
-  &lt;/DataToolbarGroup&gt;
-</pre>
-
-4) Two props can be optionally added to the top level `DataToolbar` component to improve user experience when using a `DataToolbarFilter`.
-First, if a `clearAllFilters` event handler is passed to `DataToolbar`, a 'Clear All Filters' action will appear alongside the applied filters chip groups.
-Second, if a `collapseListedFiltersBreakpoint` is passed to `DataToolbar`, the chip groups will collapse to a summary message at the passed in breakpoint, making the applied filters chip groups more responsive.
-Add the following code to the `DataToolbar` to see optional props in action.
-
-<pre class="file" data-target="clipboard">
-  clearAllFilters={this.onDelete} collapseListedFiltersBreakpoint=&quot;xl&quot;
-</pre>
-
-Take some time to play with the toolbar to see how it responds to changes in viewport size and the number of filters applied.
+&lt;DemoTable nameFilters={filters.name} statusFilters={filters.status} riskFilters={filters.risk} clearAllFilters={this.onDelete}/&gt;
+</pre> 
 
-<img src="toolbar-filter/assets/wide-view-with-chip-groups.png" alt="Wide view of toolbar filter with applied filters in chip groups" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+The `DemoTable` component is defined in `src/components/table.js` where the managed list of applied filters manipulates a very small set of data.
```

Comment:
Should this information be moved up underneath the step, or is this more of a "Note"

Reply:
It's more of a note. I'll add **Note:** before it.